packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -252,14 +252,16 @@ export default class MigrateVmsModalBody extends BaseComponent {
     })
   }
 
-  compareContainers = (pool1, pool2) => {
-    let comparePoolsId = null
-    forEach(this.props.vms, vm => {
-      comparePoolsId =
-        pool1.id === vm.$poolId ? -1 : pool2.id === vm.$poolId ? 1 : 0
-    })
-    return comparePoolsId
-  }
+  compareContainers = createSelector(
+    () => this.props.vms,
+    vms => {
+      const selectedVms = []
+      forEach(vms, vm => selectedVms.push(vm))
+      return createCompare([
+        pool => selectedVms.some(vm => vm.$pool === pool.id),
```

Comment:
You can use [lodash/some](https://devdocs.io/lodash~4/index#some), it supports collections of type object.